plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php

```diff
@@ -1008,11 +1015,8 @@ public function cleanup_post_data( array $args = array(), array $assoc_args = ar
 	 * Displays a summary of HPOS situation on this site.
 	 *
 	 * @since 8.6.0
-	 *
-	 * @param array $args       Positional arguments passed to the command.
-	 * @param array $assoc_args Associative arguments (options) passed to the command.
 	 */
-	public function status( array $args = array(), array $assoc_args = array() ) {
```

Comment:
Do you think changing the signature here will cause any issues? I suppose not unless someone is extending this class?

Reply:
Not sure if this class is truly being inherited somewhere, but better safe than sorry. I've restored the args and added a PHPCS exception. Thanks V!